docs/src/develop.md

```diff
@@ -0,0 +1,58 @@
+# Development notes
+
+## Set up
+
+Clone SpeedyWeather.jl from GitHub and change the current working directory:
+
+```
+git clone https://github.com/milankl/SpeedyWeather.jl.git
+cd SpeedyWeather.jl
+```
+
+To install Julia, manually download the latest version from the [Julia release page](https://julialang.org/downloads/) or, alternately, install it using [juliaup](https://github.com/JuliaLang/juliaup), the official Julia installer and version multiplexer. For Linux, macOS, and WSL run:
+
+```
+curl -fsSL https://install.julialang.org | sh
+```
+
+
+## Install
+
+To install SpeedyWeather.jl as a development package and [Revise.jl](https://github.com/timholy/Revise.jl) to automatically update function definitions in a running session, run:
+
+```
+julia -e 'import Pkg; Pkg.develop(path="."); Pkg.add("Revise")'
+```
```

Comment:
I personally don't use Revise.jl, to update function definitions I execute them in VS Code. If we want to give people advice here then maybe we can choose a more inclusive language, pointing them to different possibilities they have?